File: elastiknn-plugin/src/test/java/org/elasticsearch/plugin/elastiknn/ElastiknnClusterIT.java
package org.elasticsearch.plugin.elastiknn;

import org.apache.http.HttpHost;
import org.elasticsearch.test.ESIntegTestCase;
import org.junit.Before;

import java.io.IOException;

//@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.SUITE, numClientNodes = 1)
public class ElastiknnClusterIT extends ESIntegTestCase {

    @Before
    public void setup() {
        ensureGreen();
    }

    public void testDummy() {
        assertTrue(true);
    }

    public void testPluginInstalled() throws IOException {
        this.ensureGreen();
        HttpHost host = ESIntegTestCase.getRestClient().getNodes().get(0).getHost();
        assertTrue(true);
    }

}

File: elastiknn-plugin/src/test/scala/org/elasticsearch/plugin/elastiknn/ElastiknnClusterIT.scala
package org.elasticsearch.plugin.elastiknn

import java.util
import java.util.Collections

import org.elasticsearch.plugins.Plugin
import org.elasticsearch.test.ESIntegTestCase
import org.junit.Before

class ElastiknnClusterIT extends ESIntegTestCase {

  @Before
  override def setUp(): Unit = {
    this.ensureGreen()
    super.setUp()
  }

  override def nodePlugins(): util.Collection[Class[_ <: Plugin]] =
    Collections.singletonList(classOf[ElastiKnnPlugin])

  def testDummy(): Unit = assert(true)

}
